### ISSUE TITLE
Export to REP format

### ISSUE THREAD

**Issue by IanMayo** [debrief/pepys-import#825]
## 🐞 Overview

Sometimes analysts wish to send data to other agencies in a standard format.  Sadly there isn't a standard format.  The Debrief format is relatively mature that the users are familiar with.

The format is documented here:
https://debrief.github.io/tutorial/reference.html#replay_track_format

Three types of measurement can be exported to REP format: States, Contacts and Measurements.

If Maintainers view is showing any of these tables, and one or more rows are selected then "Export to REP format" action will become enabled.

A dialog will open allowing the path/filename to be specified.  The default path will be `~\Documents\` and the filename will be the current date-time, expressed as `yymmdd-hhmmss`.  Contacts data will have a `dsf` (Debrief Sensor Format) suffix, states and comments will have a `.rep` suffix.

### States
Not all data is known for the States export - particularly the symbology.  But we can make some deductions:
- the vessel name will be made from the platform name and identifier (in double quotes)
- if the States are all for one platform, then examine the platform-type and see if there is corresponding entry in the [Symbol codes](https://debrief.github.io/tutorial/reference.html#debrief_Sym).
- we don't know if the platform is blue or red. So, make it Orange so the analysts can see which platforms have not been assigned to the correct force.
- the label field is not required.

### Contacts
These will be stored in the `SENSOR:` format.  No sensor location will be provided. The ownship name will be deduced from Sensor/Platform, and constructed from name and identifier (as above). No label is required.

### Comments
If a `comment-type` is present for a comment then it will be exported as `NARRATIVE2:`, else `NARRATIVE:`. The name and identifier will be used as track name.

## 🔗 Feature

#600

**Comment by robintw** [comment]
When you say:

> Three types of measurement can be exported to REP format: States, Contacts and Measurements.

Do you mean States, Contacts and Comments?

**Comment by robintw** [comment]
We also already have some of this implemented under the `Export` option in Pepys Admin - we can export a Datafile to REP - so we should be able to make some of this code more generic and then use it for exporting from the GUI too.

**Comment by IanMayo** [comment]
Yes - exporting to file from Maintenance Page is a good solution.

I can confirm that we export States, Contacts and Comments to REP format already in the high level `Export` page.
